multi-agent/lib/mas/core/execution_context.py

```
@@ -80,6 +82,21 @@ def credential_user_id(self) -> str:
             return ""
         return self.identity.id
 
+    def with_hitl(self, enabled: bool) -> ExecutionContext:
```

Ruff (0.15.20)

[warning] Boolean-typed positional argument in function definition

(FBT001)